substrate/frame/assets/src/lib.rs

```diff
@@ -133,6 +142,8 @@
 //!
 //! * [`System`](../frame_system/index.html)
 //! * [`Support`](../frame_support/index.html)
+//!
+//! [`tokens_in_substrate`]: ../polkadot_sdk_docs/reference_docs/tokens_in_substrate/index.html
```

Comment:
I didn't know you could link like that, sweet

Reply:
Yeah, it's nice for when clippy would otherwise auto-wrap the link.